aya/src/programs/links.rs

```diff
@@ -81,7 +81,12 @@ impl<T: Link> Drop for LinkMap<T> {
 #[derive(Debug, Hash, Eq, PartialEq)]
 pub struct FdLinkId(pub(crate) RawFd);
 
-/// A file descriptor link.
+/// FdLink is a generic file descriptor link.
```

Comment:
"A file descriptor link." seems better to me for consistency with the rest of the docs. We don't repeat the type name, eg:

```
// A kernel probe.
...
struct KProbe {
```

I'd say:

```
/// A file descriptor link.
/// 
/// Fd links are returned directly when attaching some program types (for
/// instance [`CgroupSkb`]), or can be obtained by converting other link
/// types (see the `TryFrom` implementations).
/// 
/// An important property of fd links is that they can be pinned. Pinning
/// can be used keep a link attached "in background" even after the program
/// that has created the link terminates.
///
/// # Example
/// ...
```

Reply:
Done!